### PR TITLE
fix(streaming): enable tracing for executors under release profile

### PR DESCRIPTION
**Pull request by BugenZhao**
Signed-off-by: Bugen Zhao <i@bugenzhao.com>I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

- enable tracing for streaming executors under release profile (forgotten in #10315)
- enable "release-verbose" await-tree option for RiseDev by default

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Documentation

- [ ] My PR contains user-facing changes.

<details><summary>Click here for Documentation</summary>

### Types of user-facing changes

Please keep the types that apply to your changes, and remove the others.

- Installation and deployment
- Connector (sources & sinks)
- SQL commands, functions, and operators
- RisingWave cluster configuration changes
- Other (please specify in the release note below)

### Release note

</details>

### CODE REVIEW

**Review by yezizp2012 — APPROVED** [review]

File: src/stream/src/executor/wrapper/trace.rs [hunk 1]
```diff
@@ -48,9 +48,9 @@ pub async fn trace(
             if chunk.cardinality() > 0 {
                 metrics
                     .executor_row_count
-                    .with_label_values(&[&actor_id_string, &info.identity])
+                    .with_label_values(&[&actor_id_string, &span_name])
```
Comment (wcy-fdu) [inline src/stream/src/executor/wrapper/trace.rs:51]: span_name seems to contain strange numbers(`100000000C`), is this expected behavior?
Or we can remove this info from grafana dashboard.
image

Comment (BugenZhao) [reply src/stream/src/executor/wrapper/trace.rs:51]: This is the executor id in hex format. Not sure if it is useful for debugging.

Comment (wcy-fdu) [reply src/stream/src/executor/wrapper/trace.rs:51]: I notice that we have show executor id outside it , and maybe it's outdated. I have try to remove `executor_id` from identity, but too many files changed🥲. Let's keep this for now.

Comment (BugenZhao) [reply src/stream/src/executor/wrapper/trace.rs:51]: The id we shown "outside" is actor id and operator id, while the executor id is a combination of them. But it's still right that a pair of actor id and operator id is enough for locating an executor.

Comment (BugenZhao) [reply src/stream/src/executor/wrapper/trace.rs:51]: BTW, the identity should actually be the `Display` format of the corresponding frontend plan node. See https://github.com/risingwavelabs/risingwave/issues/5940.

**Pull request merged**